pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-reconciler.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: customer-admin-reconciler
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+  - create
```

Comment:
doesn't need create, right?

Reply:
This did require create.  It would fail the `auth reconcile` if not granted.